mmcv/cnn/utils/weight_init.py

```diff
@@ -8,6 +8,7 @@
 import torch.nn as nn
 from torch import Tensor
 
+from mmcv.runner.base_module import update_init_info
```

Comment:
It is not a good choice to import runner from utils, which may cause recursive import. Why not make `update_init_info` a method of BaseModule?

Reply:
Fix it in PR #1217